File: AtCoder Problems/AtCoder Beginners Selection/ABC088B - Card Game for Two/ABC088B - Card Game for Two.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	var N int
	fmt.Scan(&N)

	aList := make([]int, N)
	for i := 0; i < N; i++ {
		fmt.Scan(&aList[i])
	}

	sort.Sort(sort.Reverse(sort.IntSlice(aList)))

	var alice, bob int

	for i := 0; i < N; i++ {
		if i%2 == 0 {
			alice += aList[i]
		} else {
			bob += aList[i]
		}
	}
	fmt.Println(alice - bob)
}

File: AtCoder Problems/AtCoder Beginners Selection/ABC088B - Card Game for Two/ABC088B - Card Game for Two.py
N = int(input())
A_LIST = list(map(int, input().split()))
alice = 0
bob = 0

A_LIST.sort(reverse=True)

for i in range(N):
  if i % 2 == 0:
    alice += A_LIST[i]
  else:
    bob += A_LIST[i]

print(alice - bob)